tests/unit/s2n_server_key_share_extension_test.c

```diff
@@ -251,6 +251,7 @@ int main(int argc, char **argv)
                 struct s2n_connection *client_conn = NULL;
 
                 EXPECT_NOT_NULL(client_conn = s2n_connection_new(S2N_CLIENT));
+                EXPECT_SUCCESS(s2n_config_set_cipher_preferences(client_conn->config, "default_tls13"));
```

Comment:
Setting the cipher preferences of this config is modifying the global default config, which may impact subsequent tests. A new config should be set on the connection in order to configure it, or the policy can just be set on the connection:
```suggestion
                EXPECT_SUCCESS(s2n_connection_set_cipher_preferences(client_conn, "default_tls13"));
```